test/test.coffee

```diff
@@ -69,20 +72,28 @@ describe 'calendar plugin', ->
 			expect(mock.el).to.have.key 'radarData'
 
 		it 'uses the labels as keys in the radarData', ->
-			expect(mock.el.radarData()).to.have.key label
+			expect(mock.el.radarData()).to.have.key 'Starts Now'
 
-		it 'puts the days since the Epoch into the values in the radarData', ->
+		it 'puts the distance from the Epoch into the values in the radarData', ->
 			data = mock.el.radarData()
-			expect(data[label]).to.have.key 'value'
-			expect(data[label].value).to.eql 16679
+			expect(data['Starts Now']).to.have.key 'value'
+			expect(data['Starts Now'].value).to.eql 16679
 
-		it 'uses "weeks" as the units of the values in the radarData', ->
+		it 'specifies units & precision with values in the radarData', ->
 			data = mock.el.radarData()
-			expect(data[label]).to.have.key 'units'
-			expect(data[label].units).to.eql ['days']
-	
+			expect(data['Starts Now']).to.have.key 'units'
+			expect(data['Starts Now'].units).to.eql ['day']
+			expect(data['Starts Now']).to.have.key 'precision'
+			expect(data['Starts Now'].precision).to.eql 86400000
+
+		it 'chooses units & precision to match the parsed span of the date', ->
+			data = mock.el.radarData()
+			expect(data['Some languages were born']).to.have.key 'units'
+			expect(data['Some languages were born'].units).to.eql ['decade']
+			expect(data['Some languages were born']).to.have.key 'precision'
+			expect(data['Some languages were born'].precision).to.eql 315576000000.0
```

Comment:
So, here... more thinking about precision (as referenced in the comment in the previous test)

I don't remember any good reason for using an `int` there and a `float` here.  Think maybe this one should be an `int` too.

I also think these numbers are lacking an expressive name.  Maybe this needs something like

```
precisionForDECADE = Math.round(1000 * 60 * 60 * 24 * 365.25 * 10)
expect(data['Some languages were born'].precision).to.eql precisionForDECADE
```

I can never decide how I feel about duplicating in a test the value of a constant that I've also defined in the subject under test.  I don't like the duplication, but I do like names that express the intent better than just a bare integer.

(I guess I think it's a feature that today I get to be my future self reflecting on the work of my past self)